proposals/all_proposals.md

```diff
@@ -0,0 +1,671 @@
+
+Female Tech, Woman in Technology, Diversity
+
+Women are grossly underrepresented in tech jobs. At least in western countries. However in countries of the former socialist block there are many more women graduating from technical studies as well as working in tech jobs. This is reporting amongst others by dy the study dean 
+of the faculty of informatics at the Technical University of Vienna Hilda Tellioglu and Maria Zesch former CEO of T-Mobile Croatia. Part of the reason might be inadequate methods of teaching especially in mathematics. I know of Bostonian families who send their kids to schools teaching mathematics with methodology and curricula imported from Russia.
+
+
+
+I am not aware of studies about the reasons of the gender gap in tech. If there are none 
+
+- studies need to be carried out to find out the reasons in the teaching methods for the  gender gap in tech
+- improvement of tech teaching methodology from grammar schools to universities 
+with special gender bias and encouraging und promoting girls and women
+- initiating programs like "Female Entrepreneurship Initiative Programme"
+
+Make Open Source mandatory in governmental IT
+
+There are three main reasons for using Open Source software:
+- independence from non-European suppliers
+- security reasons with unknown content of proprietary software
+- develop local know how and create local revenue
+
+# 9 
+# 05.07.2018 Questionnaire submitted by Diego Naranjo (DiEM25 Belgium, NC member and Anna Mazgal) - 10-07-05
+
+Questions:
+
+Democratising technologies
+
+1. Can you name practices and problems in any technology that you and others see as especially "undemocratic"?
+
+There are a number of problems in relation with technology that have existed either since the emergence of a given technology (insecure mobile infrastructure) or that have evolved in an unsustainable way (from the Open Internet to silos of privately owned social networks). Some specific problems in different fields are:
+
+Data Protection and Privacy:
+
+In the field of data protection and privacy, there are many issues that have been on the spotlight in recent years: The Snowden revelations and some of the Wikileaks leaks revealed the magnitude of surveillance by public and private actors. Scandals at Facebook (most recently the Cambridge Analytica usage of personal data enabled by Facebook) brought into the light problems of profiling for political purposes and abuses of personal data without the knowledge of both users and non-users of these services. In the short-term, the most ambitious implementation of the General Data Protection Regulation (GDPR) should be encouraged. In the medium-term, in order to ensure a global approach, we should push non-EU states to facilitate the expansion of similar standards via the signature and ratification by all States of the Convention 108 of the Council of Europe.
```

Comment:
This assumes GDPR actually achieves the goal. See https://youbroketheinternet.org/GDPR for a criticism why GDPR may in large part fail to actually achieve the intended goals. - lynx